lottery/src/lib.rs

```diff
@@ -122,80 +123,10 @@ impl Lottery {
         }
     }
 
-    /// Removes player from lottery
-    /// Requirements:
-    /// * Lottery has started and lottery time has not expired
-    /// * Player must be on the player list
-    /// * Player can only remove himself
-    /// Arguments:
-    /// * `index`: lottery player index
-    async fn leave_lottery(&mut self, index: u32) {
-        if !self.lottery_is_on() {
-            panic!("leave_lottery(): Lottery on: {}", self.lottery_is_on());
-        }
-
-        if let Some(player) = self.players.get(&index) {
-            if player.player_id != msg::source() {
-                panic!(
-                    "leave_lottery(): ActorId's does not match: player: {:?}  msg::source(): {:?}",
-                    player.player_id,
-                    msg::source()
-                );
-            }
-
-            if self.token_address.is_some() {
-                let balance = player.balance;
-                self.transfer_tokens(&exec::program_id(), &msg::source(), balance)
-                    .await;
-            } else {
-                msg::send_bytes(player.player_id, b"LeaveLottery", player.balance).unwrap();
-            }
-
-            self.players.remove(&index);
-        } else {
-            panic!("leave_lottery(): Player {} not found", index);
-        }
-    }
-
-    /// Gets the player's balance
-    /// Requirements:
-    /// * Only owner can request a balance
-    /// * Lottery has started and lottery time has not expired
-    /// * Player must be on the player list
-    /// Arguments:
-    /// * `index`: lottery player index
-    fn get_balance(&mut self, index: u32) {
-        if msg::source() == self.lottery_owner && self.lottery_is_on() {
-            if let Some(player) = self.players.get(&index) {
-                msg::reply(LtEvent::Balance(player.balance), 0).unwrap();
-            } else {
-                panic!("get_balance(): Player {} not found", index);
-            }
-        } else {
-            panic!("get_balance(): Lottery on: {}", self.lottery_is_on());
-        }
-    }
-
-    /// Gets a list of players
-    /// Requirements:
-    /// * Lottery has started and lottery time has not expired
-    /// * List of players must not be empty
-    fn get_players(&mut self) {
-        if self.lottery_is_on() && !self.players.is_empty() {
-            msg::reply(LtEvent::Players(self.players.clone()), 0).unwrap();
-        } else {
-            panic!(
-                "get_players(): Lottery on: {}  players.is_empty(): {}",
-                self.lottery_is_on(),
-                self.players.is_empty()
-            );
-        }
-    }
-
     // Random number generation from block timestamp
     fn get_random_number(&mut self) -> u32 {
         let timestamp: u64 = exec::block_timestamp();
-        let code_hash = sp_core_hashing::blake2_256(&timestamp.to_be_bytes());
+        let code_hash = blake2_256(&timestamp.to_be_bytes());
```

Comment:
Idiomatic call of extern functions is:
```suggestion
        let code_hash = hashing::blake2_256(&timestamp.to_be_bytes());
```

Reply:
Fixed